atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1900,6 +1900,35 @@ private void putMetadataWithoutChangingSettings(final TableReference tableRef, f
         delete(AtlasDbConstants.DEFAULT_METADATA_TABLE, oldVersions);
     }
 
+
+    @Override
+    public void deleteRange(final TableReference tableRef, final RangeRequest range) {
+        if (range.equals(RangeRequest.all())) {
+            try {
```

Comment:
Can we just call truncateTables() here, so we know the execution logic is exactly the same?